src/util/importROAM.ts

```diff
@@ -0,0 +1,37 @@
+import { SimplePath, Timestamp } from '../types'
+import { State } from './initialState'
+import { importJSON } from './importJSON'
+import { Block } from '../action-creators/importText'
+import { RoamBlock, RoamPage } from 'roam'
+import { timestamp } from './timestamp'
+
+/**
+ * Recursively converts the roam children to blocks.
+ */
+const convertRoamBlocksToBlocks = (children: RoamBlock[]): Block[] => children.map(({ string, children, 'edit-time': editTime, 'create-time': createTime }) => ({
+  scope: string,
+  created: new Date(createTime).toISOString() as Timestamp,
+  lastUpdated: editTime ? new Date(editTime).toISOString() as Timestamp : timestamp(),
+  children: children && children.length ? convertRoamBlocksToBlocks(children) : []
+}))
+
+/**
+ * Converts the ROAM JSON to an array of blocks.
+ */
+const roamJsontoBlocks = (ROAM: RoamPage[]) => {
```

Comment:
Please change all instances of `ROAM` to `roam` as appropriate. This was requested in my previous review. You should also rename `ROAM` to `Roam` in the comments.